Source/Other/Extensions/NSDictionaryExtension.swift

```diff
@@ -16,9 +16,13 @@ import Foundation
         let font = UserDefaults.getFont()
         let attributedString = NSMutableAttributedString(string: columnName, attributes: [.font: font])
         if let columnType: String = value(forKey: "type") as? String {
-            attributedString.append(NSAttributedString(string: " "))
+            attributedString.append(NSAttributedString(string: NSDictionary.tableContentColumnHeaderSplittingSpaceCharacter as String))
             attributedString.append(NSAttributedString(string: columnType, attributes: [.font: NSFontManager.shared.convert(font, toSize: 8), .foregroundColor: NSColor.gray]))
         }
         return attributedString
     }
+    static var tableContentColumnHeaderSplittingSpaceCharacter: NSString {
```

Comment:
Let's put this into `StringExtension` as

`@objc static let columnHeaderSplittingSpace: String = " "` instead of onto NSDictionary

Reply:
It looks like Xcode doesn't like that because String isn't a class in Objective-C? Am I missing something here?

Reply:
Ope, figured it out - StringExtension has both String and NSString 🤦

Reply:
Yeah we have both for now :/